<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingsUI</class>
 <widget class="QDialog" name="settingsUI">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>301</width>
     <height>32</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close|QDialogButtonBox::Save</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>311</width>
     <height>271</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabGeneral">
    <attribute name="title">
     <string>General</string>
    </attribute>
    <widget class="QLabel" name="lbServerIp">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Server IP address</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="serverAddress">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>100</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbServerLoopback">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>80</y>
       <width>87</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Loopback</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbNetwork">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="text">
      <string>Network</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="repeater_id">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>130</y>
       <width>48</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="toolTip">
      <string>Brandmeister extended ID</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLabel" name="lbRepeaterid">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>132</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Brandmeister extended ID</string>
     </property>
     <property name="text">
      <string>Repeater ESSID</string>
     </property>
    </widget>
    <widget class="QComboBox" name="startup_mode">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>190</y>
       <width>111</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="toolTip">
      <string>Startup mode</string>
     </property>
     <item>
      <property name="text">
       <string>DMR</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DStar  (Icom, Kenwood)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NXDN (Kenwood, Icom)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>P25     (Motorola, Qantar)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>YSF     (Yaesu Fusion)</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="lbStartupMode">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Startup mode</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="mycall">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>4</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="toolTip">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbCall">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>4</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>My Call</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="mydmr_id">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>33</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="toolTip">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbUsrpPort">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>USRP port</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="usrp_port">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>160</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="toolTip">
      <string>DVSwitch connection port</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbDmrid">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>33</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>DMR ID</string>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>60</y>
       <width>295</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabAudio">
    <attribute name="title">
     <string>Audio</string>
    </attribute>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>90</y>
       <width>295</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="rxLevel">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>200</y>
       <width>210</width>
       <height>22</height>
      </rect>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="lbTxAudio">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>135</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Tx</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbDongleMode">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>61</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Dongle mode</string>
     </property>
    </widget>
    <widget class="QComboBox" name="rxOutput">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>175</y>
       <width>240</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
    </widget>
    <widget class="QComboBox" name="txInput">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>125</y>
       <width>240</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>31</y>
       <width>236</width>
       <height>26</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lbThreshold">
        <property name="text">
         <string>Threshold</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="voxThreshold">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
        <property name="value">
         <number>200</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbDelay">
        <property name="text">
         <string>Delay</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="voxDelay">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="maximum">
         <number>500</number>
        </property>
        <property name="value">
         <number>50</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QSlider" name="txLevel">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>150</y>
       <width>210</width>
       <height>22</height>
      </rect>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="lbVox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Vox</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbRxAudio">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>175</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Rx</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbVoxEnable">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Enable</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbAudio">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Audio</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabOptions">
    <attribute name="title">
     <string>Options</string>
    </attribute>
    <widget class="QCheckBox" name="cbQrzLookup">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>QRZ lookup</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbMinToTray">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>185</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Minimize to tray</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbPttToggle">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>35</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Ptt toggle (on/off)</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="pttTimeout">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Timeout in seconds</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="suffix">
      <string> secs</string>
     </property>
     <property name="maximum">
      <number>600</number>
     </property>
     <property name="value">
      <number>200</number>
     </property>
    </widget>
    <widget class="QLabel" name="lbTxTimeout">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>60</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Tx timout</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbImportConfig">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>210</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Import Config</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbExportConfig">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>210</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Export Config</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbShortCalls">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Log short calls ( &lt; 1sec)</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="cbTheme">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>127</y>
       <width>161</width>
       <height>26</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>System</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Light</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dark</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="lbTheme">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>131</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Theme</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbLoglevel">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>158</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Log level</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbLoglevel">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>155</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Debug</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Info</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Warning</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Error</string>
      </property>
     </item>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>110</y>
       <width>295</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
